postgresql/src/main/java/org/apache/zeppelin/postgresql/PostgreSqlInterpreter.java

```diff
@@ -264,7 +286,13 @@ public Scheduler getScheduler() {
 
   @Override
   public List<String> completion(String buf, int cursor) {
-    return null;
+
+    List<CharSequence> candidates = new ArrayList<CharSequence>();
+    if (sqlCompleter.complete(buf, cursor, candidates) >= 0) {
+      return Lists.transform(candidates, sequenceToStringTransformer);
+    } else {
+      return NO_COMPLETION;
```

Comment:
i can not move it because the `candidates` is used inside the `if` statement. It is the IN/OUT argument.  The signature is defined by the JLine Completer API. 
But perhaps you meant something else?